lighthouse-core/scripts/lantern/collect/README.md

```diff
@@ -0,0 +1,17 @@
+# Lantern Collect Traces
+
+## Get a WPT key
+
+http://www.webpagetest.org/getkey.php -> "Login with Google" -> fill form. Key will be emailed to you.
+
+## Lighthouse Version
+
+Check what version of Lighthouse WPT is using. You should use the same version of lighthouse for the desktop collection.
```

Comment:
so: i'll collect the LHR, devtoolslog, and trace for local unthrottled desktop runs. And the LHR and trace for WPT runs. put them all in a giant zip and upload somewhere.

then make the new golden lantern set by taking the median runs for the local runs. and just run trace-processor on the WPT traces to get the metrics.

Reply:
right-o, but since we don't have LCP metric yet, I would triple check that the traces you're getting have the LCP and layout stability events in them at least and we don't need to enable some other category to get them.

Reply:
yup they do